test/server-side-rendering/samples/head-html-and-component/HeadNested.svelte

```diff
@@ -0,0 +1,2 @@
+{@html '<meta name="head_nested_html" content="head_nested_html">'}
+<meta name="head_nested" content="head_nested">
```

Comment:
can add another case where `HeadNested` contains another `<svelte:head>`, which gives you the nested `<!-- HEAD_START -->` comment

Reply:
@tanhauhau,

I have analyzed the nested `<svelte:head>` case and currently it not feasible, even in the latest release of svelte.
The content of the 2nd level `<svelte:head>` ends up as an empty string.

This is due to how the SSR html is generated. [REPL example](https://svelte.dev/repl/3a0d30d3a33248aea94d47be8afcd0f0?version=3.49.0).

An equivalent example of what is compiled by svelte:
```
const $$result = { head: '' };

function $$render(result) {
    result.head += '_NESTED_';
    return '';
}

$$result.head += `_PREFIX_ ${$$render($$result)} _SUFFIX_`;

console.log($$result.head); // Logs "_PREFIX_  _SUFFIX_"
```javascript

To support this, I imagine a sensitive change should be done to the compiler.
Probably is out of scope of this task because it is an existent limitation?

Let me know your thoughts!

Thanks

Reply:
yea i think the current implementation is good. 

i think we just need to add it as a new test case to make sure this get hydrated correctly too.

Reply:
Did you mean this test or a new one?

This test is already replicated at `test/hydration/samples/head-html-and-component/main.svelte` and test hydration.

Let me know if I miss something.

Thanks

Reply:
Did you mean this test or a new one?

This test is already replicated at test/hydration/samples/head-html-and-component/main.svelte and test hydration.

Let me know if I miss something.

Thanks

Reply:
Hi again @tanhauhau, please let me know if there is something missing so I can organize my spare time to make it happen! Thanks